core/mutator.js

```diff
@@ -166,18 +166,17 @@ Blockly.Mutator.prototype.createEditor_ = function() {
       ({
         // If you want to enable disabling, also remove the
         // event filter from workspaceChanged_ .
-        disable: false,
-        languageTree: quarkXml,
-        parentWorkspace: this.block_.workspace,
-        pathToMedia: this.block_.workspace.options.pathToMedia,
-        RTL: this.block_.RTL,
-        toolboxPosition: this.block_.RTL ? Blockly.TOOLBOX_AT_RIGHT :
+        'disable': false,
```

Comment:
Why quote the keys?

Reply:
For advanced compilation. Otherwise they’ll get renamed.